metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "hunner-wordpress",
+  "version": "1.0.0",
+  "author": "Hunter Haugen",
+  "summary": "Puppet module to set up an instance of wordpress; and optionally a mysql db/user.",
+  "license": "Apache-2.0",
+  "source": "https://github.com/hunner/puppet-wordpress",
+  "dependencies": [
+    {"name":"puppetlabs/concat",    "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/mysql",     "version_requirement":">= 2.1.0"},
+    {"name":"puppetlabs/stdlib",    "version_requirement":">= 2.3.1"}
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "OracleLinux",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "8"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "16.04"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.7.0 < 6.0.0"
+    }
+  ],
+  "pdk-version": "1.4.1",
+  "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
+  "template-ref": "1.4.1-0-g52adbbb"
```

Comment:
I think if `--template-url` is the default then the template-ref won't be needed? https://github.com/puppetlabs/pdk/blob/v1.4.1/lib/pdk/generate/module.rb#L71-L75

Reply:
`--template-ref` is still needed for pdk update. So that if you want to resync or update your module, we have a comparison point.